airbyte-integrations/connectors/source-sendgrid/source_sendgrid/created_expected_records.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import json
+import os
+import pprint
+import sys
+from typing import List
+
+
+def list_output_files(root_dir: str) -> List[str]:
+    return os.listdir(root_dir)
+
+
+def get_expected_records_for_stream(root_dir: str, filename: str):
+    full_path = os.path.join(root_dir, filename)
+    with open(full_path, "r") as f:
+        records = f.readlines()
+        return [json.loads(r) for r in records]
+
+
+def get_stream_name_from_filename(filename: str):
+    return filename.split("_airbyte_raw_")[1].split(".jsonl")[0]
+
+
+def transform_record_to_expected_record(record, stream):
```

Comment:
also needs to filter out fields that will differ in the future.
in the case of sendgrid, we want to filter out the sampling timestamps